### DISCUSSION TITLE
Does the possibility exist to render the pdf using blazor canvas?

### DISCUSSION THREAD

**Discussion by ChinaChenMingQuan** [BobLd/PdfPig.Rendering.Skia#12] [Q&A]
Can I use the web canvas api instead of skia for the purpose of rendering a pdf page?

Does this possibility exist?

deepl translator.

**Comment by BobLd** [comment]
This might be possible, I see skiasharp has blazor components https://github.com/mono/SkiaSharp/blob/main/source/SkiaSharp.Views/SkiaSharp.Views.Blazor/SKGLView.razor.cs
